bundles/pom.xml

```diff
@@ -1,567 +1,1111 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
The formatting of this pom.xml has gone wrong due to the skeleton script. Can you revert it, and only leave your own binding in it.

Reply:
It's not specifically about those lines. But all lines are changed. Look in GitHub in your pr and you understand what I mean.Take the original pom.xml ([here is the original pom.xml](https://raw.githubusercontent.com/openhab/openhab-addons/89f5538f257ecd54e23881f1f9c10c22a50d9058/bundles/pom.xml)) and edit it to add your binding, and commit that change.

Reply:
I have reverted to the old document and added my binding. Should be OK now.